HeterogeneousCore/CUDAServices/plugins/NVProfilerService.cc

```diff
@@ -523,18 +531,8 @@ void NVProfilerService::preallocate(edm::service::SystemBounds const& bounds) {
   }
 }
 
-void NVProfilerService::preBeginJob(edm::PathsAndConsumesOfModulesBase const& pathsAndConsumes,
-                                    edm::ProcessContext const& context) {
-  callgraph_.preBeginJob(pathsAndConsumes, context);
+void NVProfilerService::preBeginJob(edm::ProcessContext const& context) {
   nvtxDomainMark(global_domain_, "preBeginJob");
-
-  // this assumes that preBeginJob is not called concurrently with the modules' beginJob method
-  // or the preBeginJob for a subprocess
-  unsigned int modules = callgraph_.size();
```

Comment:
This was the only use of the `callgraph_`.
If it is no longer needed to get the correct number of modules (because it is already known in `preallocate()`), the `callgraph_` object can be removed.

Reply:
I removed callgraph_. That's better. Thanks.